jest.conf/testUtils.js

```diff
@@ -10,6 +10,17 @@ export const resizeWindow = (width, height = 768) => {
   });
 };
 
+export function takeScreenshot() {
```

Comment:
It seems like this method is used to check if we can take a screenshot isnt it? So probably a better naming like "canTakeScreenshot" could work better, because "takeScreenshot" sounds like this method actually takes the screenshot

Reply:
Sure, I'll add the changes in my next commit.